helm/hpcc/values.schema.json

```diff
@@ -873,6 +888,9 @@
           "type": "object",
           "additionalProperties": { "type": "string" }
         },
+        "auth": {
+          "$ref": "#/definitions/auth"
+        },
```

Comment:
Again, Unless you are going to do your change to move it to the ldap section, add ldapAddress here, otherwise there is no way of setting it.